rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -57,6 +58,11 @@
     private static final String MINER_COINBASE_SECRET_CONFIG = "miner.coinbase.secret";
     private static final int CHUNK_SIZE = 192;
 
+    // Prune default values
+    private static final int PRUNE_BLOCKS_TO_COPY_DEFAULT = 5000;
```

Comment:
The default values are not needed because they're taken from `reference.conf`. Then it's just a matter of calling `configFromFiles.getInt("property.path")` because it will always be defined.